diff --git a/.npmignore b/.npmignore
deleted file mode 100644
index 4c1cebd66058632ace869cc0f1c8d3bdb1079c54..0000000000000000000000000000000000000000
diff --git a/android/build.gradle b/android/build.gradle
index cd4d6cd073204c2a6d21e1f596eebf90ee178fb4..e6df8403a5339803fac9e2d7851f98034b7e2c16 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,23 +1,16 @@
+apply plugin: 'com.android.library'
 
-buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
-    }
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
-apply plugin: 'com.android.library'
-
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion safeExtGet("compileSdkVersion", 34)
+    namespace "com.reactlibrary"
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
+        minSdkVersion safeExtGet("minSdkVersion", 23)
+        targetSdkVersion safeExtGet("targetSdkVersion", 34)
         versionCode 1
         versionName "1.0"
     }
@@ -31,9 +24,8 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
-    compile 'com.jcraft:jsch:0.1.54'
-    compile 'com.fasterxml.jackson.core:jackson-core:2.9.0'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.9.0'
+    implementation 'com.facebook.react:react-native:+'
+    implementation 'com.jcraft:jsch:0.1.54'
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.9.0'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.0'
 }
-  
\ No newline at end of file
diff --git a/android/src/main/AndroidManifest.xml b/android/src/main/AndroidManifest.xml
index 558f382b654e8631501c42111faac8e46f4dcb16..0a0938ae37e5da80351f4f988163f59d1eacf6fc 100644
--- a/android/src/main/AndroidManifest.xml
+++ b/android/src/main/AndroidManifest.xml
@@ -1,5 +1,3 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.reactlibrary">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
 </manifest>
-  
\ No newline at end of file
diff --git a/android/src/main/java/com/reactlibrary/RNSshClientModule.java b/android/src/main/java/com/reactlibrary/RNSshClientModule.java
index a8d97ac2fef135046aeeaa3a774a553fd1ad09db..3fae2b12d21b530853a22c447ad889a320a37c8d 100644
--- a/android/src/main/java/com/reactlibrary/RNSshClientModule.java
+++ b/android/src/main/java/com/reactlibrary/RNSshClientModule.java
@@ -2,7 +2,7 @@ package com.reactlibrary;
 
 import android.os.Environment;
 import android.util.Log;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
 
 import com.facebook.react.bridge.Arguments;
 import com.facebook.react.bridge.Callback;
diff --git a/example/.npmignore b/example/.npmignore
deleted file mode 100644
index e5196cff7d8803b0cf45df9f258894e7a1a8a3a1..0000000000000000000000000000000000000000
